src/trace-plugin-loader.js

```diff
@@ -197,19 +147,6 @@ function activate(agent) {
 }
 
 function deactivate() {
-  for (var moduleName in plugins) {
```

Comment:
Is this combining the `stop` changes alongside other improvements? I feel like the removal of `unpatch` would better align with that PR. It's fine to leave it here but I am curious if it is needed outside of the context of the `stop` changes.